build.ps1

```diff
@@ -22,26 +19,151 @@ function ZipContent([string] $sourceDirectory, [string] $target)
   [IO.Compression.ZipFile]::CreateFromDirectory($sourceDirectory, $target)
 }
 
+function CrossGen([string] $runtime, [bool] $isSelfContained, [string] $publishTarget, [string] $privateSiteExtensionPath)
+{
+    Write-Host "publishTarget: " $publishTarget
+    Write-Host "privateSiteExtensionPath: " $privateSiteExtensionPath
+
+    $selfContained = Join-Path $publishTarget "self-contained"
+    $regular = Join-Path $publishTarget "regular"
```

Comment:
Can you explain what "self-contained" vs "regular" means?

Reply:
Self-contained deployment contains all NET Core and ASP NET Core resources required for the application.
More info: https://docs.microsoft.com/en-us/dotnet/core/deploying/